fhirstarter/fhirstarter.py

```diff
@@ -447,37 +460,32 @@ def openapi(self) -> dict[str, Any]:
 
                 # For search operations, provide a bundle example that contains the correct resource
                 # type
-                if "|" in (
-                    operation_id := operation["operationId"]
-                ) and operation_id.startswith("fhirstarter|"):
-                    _, _, interaction_type, *rest = operation_id.split("|")
-                    if interaction_type == "search-type":
-                        resource_type = rest[1]
-
-                        # Get the example for the resource. If a custom example is defined, it will
-                        # be used, otherwise an example will be loaded from the FHIR specification.
-                        try:
-                            example = self._capabilities[resource_type][
-                                "search-type"
-                            ].resource_type.Config.schema_extra["example"]
-                        except (AttributeError, KeyError):
-                            example = None
-                        if not example:
-                            if is_resource_type(resource_type):
-                                example = load_example(resource_type)
-                            else:
-                                example = {"resourceType": resource_type}
-
-                        # For successful responses, copy the schema, and create and set a bundle
-                        # example that includes the example resource
-                        operation["responses"]["200"]["content"][
-                            "application/fhir+json"
-                        ]["schema"] = deepcopy(
-                            openapi_schema["components"]["schemas"]["Bundle"]
-                        )
-                        operation["responses"]["200"]["content"][
-                            "application/fhir+json"
-                        ]["schema"]["example"] = create_bundle_example(example)
+                _, _, interaction_type, *rest = operation_id.split("|")
```

Comment:
The changes on these lines is just indentation. The operation ID check was moved to the top of the for loop.